pkgs/applications/misc/vue/default.nix

```diff
@@ -1,50 +1,25 @@
-x@{builderDefsPackage
-  , jre, unzip
-  , ...}:
-builderDefsPackage
-(a :
-let
-  helperArgNames = ["stdenv" "fetchurl" "builderDefsPackage"] ++
-    [];
+{ stdenv, fetchurl, jre, unzip }:
 
-  buildInputs = map (n: builtins.getAttr n x)
-    (builtins.attrNames (builtins.removeAttrs x helperArgNames));
-  sourceInfo = rec {
-    baseName="vue";
-    version="3.2.2";
-    name="${baseName}-${version}";
-    url="releases.atech.tufts.edu/jenkins/job/VUE/64/deployedArtifacts/download/artifact.2";
-    hash="0sb1kgan8fvph2cqfxk3906cwx5wy83zni2vlz4zzi6yg4zvfxld";
+stdenv.mkDerivation rec {
+  name = "vue-${version}";
+  version = "3.2.2";
+  src = fetchurl {
+    url = "releases.atech.tufts.edu/jenkins/job/VUE/64/deployedArtifacts/download/artifact.2";
```

Comment:
Ach, that is vexing, I skipped trying to build vue myself as its just a 1:1 translation ... Will look into this and push a fixed version shortly. Thanks